experiments/prompt-injection-boundary-tags/rounds/round1/report/charts/generate_charts.py

```
@@ -0,0 +1,139 @@
+def save_condition_comparison(
+    *,
+    conds: list[str],
+    means: dict[str, float],
+    out_path: Path,
+) -> None:
+    fig, ax = plt.subplots(figsize=(7, 4))
+    vals = [means[c] for c in conds]
+    ax.bar(conds, vals, color="#4d4d4d")
+
+    ax.set_title("Round 1: Mean Score by Condition (Claude 3.5 Haiku)")
+    ax.set_ylabel("Mean score (0–3)")
```

Ruff (0.15.0)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)